sources/Z.100-201811-AnnF1.adoc

```diff
@@ -321,9 +320,9 @@ Embedded within a state are certain substructures or _background classes_ <<b-Bl
 
 The one-step transformation stem:[tau] updates the values of functions. In general, for stem:[s in S], some equivalences hold in _s_ but not in stem:[tau(s)] and vice versa. In some cases, τhas no effect, so _s_ and stem:[tau(s)] are the same.
 
```

Comment:
* Remove leading space.
* delta unicode should be changed to `stem:[Delta]`
* the letters should really be math.

Reply:
Fixed.